From 90b2552ec560d6f3806bcf94757e51bb5e675f98 Mon Sep 17 00:00:00 2001
From: Donn Seeley <donn.seeley@windriver.com>
Date: Tue, 12 Jun 2012 10:56:38 -0600
Subject: [PATCH 15/19] emgd: Use the standard version of BIT().

EMGD was defining its own redundant version of BIT(), rather than using
the one from <linux/bitops.h>.

Signed-off-by: Donn Seeley <donn.seeley@windriver.com>
---
 drivers/gpu/drm/emgd/emgd/include/pd.h |    6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/emgd/emgd/include/pd.h b/drivers/gpu/drm/emgd/emgd/include/pd.h
index 388a2af..7dc7f8f 100644
--- a/drivers/gpu/drm/emgd/emgd/include/pd.h
+++ b/drivers/gpu/drm/emgd/emgd/include/pd.h
@@ -32,6 +32,7 @@
 #ifndef _PD_H_
 #define _PD_H_
 
+#include <linux/bitops.h>
 #include <igd_pd.h>
 #include <igd_debug.h>
 
@@ -61,11 +62,6 @@
 #define FALSE  0
 #endif
 
-/* This needs to contain the unsigned long for vBIOS. */
-#ifndef BIT
-#define BIT(x) (1L<<x)
-#endif
-
 /* Macro to return:
  *    minimum, if the value < min value
  *    maximum, if the value > max value
-- 
1.7.9.7

